src/git.zsh

```diff
@@ -142,6 +142,96 @@ _fzf_complete_git() {
         return
     fi
 
+    if [[ $subcommand = 'pull' ]]; then
+        local prefix_option completing_option
+
+        local git_options_argument_required=(--cleanup --date --depth --deepen --negotiation-tip -o -s --server-option --shallow-exclude --shallow-since --strategy --strategy-option --strategy-option=diff-algorithm --upload-pack -X)
+        local git_options_argument_optional=(--gpg-sign --log --rebase --recurse-submodules -S)
+
+        if completing_option=$(_fzf_complete_git_parse_completing_option "$prefix" "$last_argument" ${(F)git_options_argument_required} ${(F)git_options_argument_optional}); then
+            if [[ $completing_option = --* ]]; then
+                prefix_option=$completing_option=
+            else
+                prefix_option=${prefix%%${completing_option[-1]}*}${completing_option[-1]}
+            fi
+        fi
+
+        case $completing_option in
+            --recurse-submodules)
+                local recurse_submodules=(yes on-demand no)
+                _fzf_complete_git_git_constants '' "${(F)recurse_submodules}" $@
+                ;;
+
+            --cleanup)
+                local cleanup_modes=(strip whitespace verbatim scissors default)
+                _fzf_complete_git_git_constants '' "${(F)cleanup_modes}" $@
+                ;;
+
+            -s|--strategy)
+                local strategies=(octopus ours subtree recursive resolve)
+                _fzf_complete_git_git_constants '' "${(F)strategies}" $@
+                ;;
+
+            --strategy-option|--strategy-option=diff-algorithm|-X)
+                local strategy_options=(
+                    diff-algorithm=histogram
+                    diff-algorithm=minimal
+                    diff-algorithm=myery
+                    diff-algorithm=patience
+                    find-rename
+                    find-rename=
+                    histogram
```

Comment:
I could not find `histogram` as a strategy option.

See: https://git-scm.com/docs/git-merge#_merge_strategies